README.md

```diff
@@ -121,8 +121,9 @@ Some payment methods need additional configuration. For example `ApplePayCompone
 
 ```swift
 let configuration = DropInComponent.Configuration(apiContext: apiContext)
-configuration.applePay.summaryItems = "..." // Apple Pay summary items.
-configuration.applePay.merchantIdentifier = "..." // Apple Pay merchant identifier.
+configuration.payment = payment
+configuration.applePay = .init(summaryItems: ConfigurationConstants.applePaySummaryItems,
+                               merchantIdentifier: ConfigurationConstants.applePayMerchantIdentifier)
```

Comment:
`ConfigurationConstants ` doesn't make sense in the read me file, can you make it `{summaryItems}` and `"{applePayMerchantIdentifier}"`?

Reply:
good point...